README.md

```diff
@@ -5,12 +5,8 @@
 
 # Canopy
 
-> An Angular implementation of the Legal & General design system.
+> We are working on the brand modernisation of the Legal & General design system.
 
-Interested in [contributing?](docs/CONTRIBUTING.md)
+Brand modernisation [storybook](https://legal-and-general.github.io/canopy/master-bm?path=/docs/welcome--docs)
```

Comment:
I think the storybook path will be: `https://legal-and-general.github.io/canopy/lg-sb-master-bm`

Reply:
OK, I will update Elena 👍